include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -122,6 +122,15 @@ typedef enum OrtErrorCode {
   ORT_REQUIREMENT_NOT_REGISTERED,
 } OrtErrorCode;
 
+typedef enum OrtExecutionProviderType {
+  ORT_EXECUTION_PROVIDER_CPU = 0,
```

Comment:
Ryan is taking an off. If he were here, he will suggest you remove the numbers in each line.
I think either way is ok, but it should be consistent.

Reply:
numbers removed.